tests/_support/Helper/ManifestHelper.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Helper;
+
+use Codeception\Module;
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+use Generated\Shared\Transfer\ManifestTransfer;
+use org\bovigo\vfs\vfsStream;
+use SprykerSdk\Zed\AppSdk\AppSdkConfig;
+use SprykerTest\Shared\Testify\Helper\ConfigHelperTrait;
+use SprykerTest\Zed\Testify\Helper\Business\BusinessHelperTrait;
+
+class ManifestHelper extends Module
+{
+    use BusinessHelperTrait;
+    use ConfigHelperTrait;
+
+    /**
+     * @var string|null
+     */
+    protected ?string $rootUrl = null;
+
+    /**
+     * @return \Generated\Shared\Transfer\ManifestRequestTransfer
+     */
+    public function haveManifestAddRequest(): ManifestRequestTransfer
+    {
+        $this->getValidatorHelper()->mockRoot($this->getRootUrl());
+
+        $config = $this->getValidatorHelper()->getConfig() ?? new AppSdkConfig();
+
+        $manifestTransfer = new ManifestTransfer();
+        $manifestTransfer
+            ->setName('Manifest3')
+            ->setLocaleName('ZZ');
```

Comment:
This is an invalid locale, you need to have a test case that covers our code when an invalid locale is passed from the user.